### PR TITLE
fix little typo, update french translation

### PR DESCRIPTION
**Pull request by Houston4444**
fix a typo in help text and update french translation.
I must say I admire the translators ! Translate html parts is particularly  boring ;)

### CODE REVIEW

File: resources/locale/qjackcapture_fr.ts [hunk 1]
```diff
@@ -296,7 +296,7 @@
     <message>
         <location filename="../ui/mainwindow.ui" line="926"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Extra command line arguments for &lt;span style=&quot; font-family:&apos;monospace&apos;;&quot;&gt;jack_capture&lt;/span&gt; command&lt;/p&gt;&lt;p&gt;Separate arguments by spaces, use shell quoting rules.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Arguments additionnels pour la commande &lt;span style=&quot; font-family:'monospace';&quot;&gt;jack_capture&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Separez les arguments par des espaces, utilisez les règles shell pour les guillemets.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Arguments additionnels pour la commande &lt;span style=&quot; font-family:&apos;monospace&apos;;&quot;&gt;jack_capture&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Separez les arguments par des espaces, utilisez les règles shell pour les guillemets.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sure for this one @Houston4444 ?

File: resources/locale/qjackcapture_fr.ts [hunk 2]
```diff
@@ -374,58 +374,66 @@
...
+        <location filename="../../qjackcapture/app.py" line="1141"/>
         <source>Current filename prefix: %s</source>
         <translation>Préfixe actuel de nom de fichier: %s</translation>
     </message>
     <message>
-        <location filename="../../qjackcapture/app.py" line="1120"/>
+        <location filename="../../qjackcapture/app.py" line="1128"/>
         <source>Current output folder: %s</source>
-        <translation type="unfinished"></translation>
+        <translation>Dossier de sortie actuel: %s</translation>
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: manque un espace avant les 2 points

File: resources/locale/qjackcapture_fr.ts [hunk 3]
```diff
@@ -374,58 +374,66 @@
...
     </message>
     <message>
-        <location filename="../../qjackcapture/app.py" line="847"/>
+        <location filename="../../qjackcapture/app.py" line="855"/>
         <source>Missing output folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Dosssier de sortie inexistant</translation>
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: un s de trop monsieur le reptilien :D

File: resources/locale/qjackcapture_fr.ts [hunk 4]
```diff
@@ -374,58 +374,66 @@
...
     <message>
-        <location filename="../../qjackcapture/app.py" line="847"/>
+        <location filename="../../qjackcapture/app.py" line="855"/>
         <source>The selected output folder does not exist:
 
 {}
 
 Create it now and proceed?</source>
-        <translation type="unfinished"></translation>
+        <translation>Le dossier de sortie sélectionné n&apos;existe pas:
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: manque un espace avant les 2 points ici aussi

File: resources/locale/qjackcapture_fr.ts [hunk 5]
```diff
@@ -374,58 +374,66 @@
...
     <message>
-        <location filename="../../qjackcapture/app.py" line="863"/>
+        <location filename="../../qjackcapture/app.py" line="871"/>
         <source>Invalid output folder:
 
 {}
 
 Please check you settings.</source>
-        <translation type="unfinished"></translation>
+        <translation>Dossier de sortie invalide:
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: manque un espace avant les 2 points

File: resources/locale/qjackcapture_fr.ts [hunk 6]
```diff
@@ -482,19 +510,19 @@ p, li { white-space: pre-wrap; }
         <translation type="obsolete">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
 &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
-&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Noto Sans&apos;; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr pour ici @Houston4444 ?

File: resources/locale/qjackcapture_fr.ts [hunk 7]
```diff
@@ -482,19 +510,19 @@ p, li { white-space: pre-wrap; }
...
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;${clientname}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - nom de client NSM ou nom de client JACK&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;${clientid}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - identifiant de client unique NSM &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;(NSM)&lt;/span&gt;&lt;br /&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;${session}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - nom de la session NSM dans laquelle ce client figure&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;(NSM)&lt;/span&gt;&lt;/p&gt;
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr ?

File: resources/locale/qjackcapture_fr.ts [hunk 8]
```diff
@@ -525,18 +553,18 @@ p, li { white-space: pre-wrap; }
...
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;C'est pratique, par exemple, lorsqu'on enregistre les ports d'entrée correspondant aux sorties matérielles, étant donné que &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; peut être lancé en premier, et qu'ensuite les autres programmes peuvent être arrêtés et démarrés à n'importe quel moment, tous les sons audibles devraient être enregistrés.&lt;/span&gt;&lt;/p&gt;
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Noto Sans&apos;; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr ?

File: resources/locale/qjackcapture_fr.ts [hunk 9]
```diff
@@ -525,18 +553,18 @@ p, li { white-space: pre-wrap; }
...
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Sources d&apos;enregistrement&lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; peut se connecter à n&apos;importe quel port JACK d&apos;entrée ou de sortie.&lt;/span&gt;&lt;/p&gt;
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr ?

File: resources/locale/qjackcapture_fr.ts [hunk 10]
```diff
@@ -525,18 +553,18 @@ p, li { white-space: pre-wrap; }
...
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; enregistrera le signal provenant des port de sortie sélectionnés (sources) ou des ports connectés aux ports d'entrée sélectionnés (sinks).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; enregistrera le signal provenant des port de sortie sélectionnés (sources) ou des ports connectés aux ports d&apos;entrée sélectionnés (sinks).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr ?

File: resources/locale/qjackcapture_fr.ts [hunk 11]
```diff
@@ -525,18 +553,18 @@ p, li { white-space: pre-wrap; }
...
 &lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Connexions manuelles:&lt;/span&gt;&lt;/p&gt;
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; ne se connectera à aucun port et vous devrez connecter ses ports d'entrée manuellement en utilisant un autre programme.&lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; ne se connectera à aucun port et vous devrez connecter ses ports d&apos;entrée manuellement en utilisant un autre programme.&lt;/span&gt;&lt;/p&gt;
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr ?

File: resources/locale/qjackcapture_fr.ts [hunk 12]
```diff
@@ -525,18 +553,18 @@ p, li { white-space: pre-wrap; }
...
 &lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Par défaut:&lt;/span&gt;&lt;/p&gt;
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; enregistrera le signal provenant de tous les ports connectés aux deux premières sorties audio physiques.&lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; enregistrera le signal provenant de tous les ports connectés aux deux premières sorties audio physiques.&lt;/span&gt;&lt;/p&gt;
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr ?

File: resources/locale/qjackcapture_fr.ts [hunk 13]
```diff
@@ -525,18 +553,18 @@ p, li { white-space: pre-wrap; }
...
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;C&apos;est pratique, par exemple, lorsqu&apos;on enregistre les ports d&apos;entrée correspondant aux sorties matérielles, étant donné que &lt;/span&gt;&lt;span style=&quot; font-family:&apos;monospace&apos;; font-size:12pt;&quot;&gt;jack_capture&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; peut être lancé en premier, et qu&apos;ensuite les autres programmes peuvent être arrêtés et démarrés à n&apos;importe quel moment, tous les sons audibles devraient être enregistrés.&lt;/span&gt;&lt;/p&gt;
```
Comment (trebmuh) [inline resources/locale/qjackcapture_fr.ts]: sûr ?

**Review by trebmuh — CHANGES_REQUESTED** [review]

**Comment by SpotlightKid** [comment]
> Translate html parts is particularly boring ;)

Are you using Qt Linguist? It makes these strings much easier to handle.

Probably you do, but just wanted to make sure ;)

**Comment by Houston4444** [comment]
> Are you using Qt Linguist? 

Of course I do, but the help texts are just full html even in Qt Linguist. So we need to take care to keep all html tags and spaces.

**Comment by Houston4444** [comment]
@trebmuh : 

J'ai désensssorcelé le reptilien, mis des espaces avant les deux points.

The rest come from updates with pylupdate5, but It doesn't seems to affect anything, btw, there was a lot of obsolete strings. Thanks for the check. If you want to test the software and see if something seems wrong in translation, you're welcome, I know you know the translation largely better than me ;)

**Comment by trebmuh** [comment]
@Houston4444 : 

> J'ai désensssorcelé le reptilien, mis des espaces avant les deux points.

:+1: :D

> If you want to test the software and see if something seems wrong in translation, you're welcome, I know you know the translation largely better than me ;)

I've got a very little time available for this king of stuff right now. I'll ping you if I find some time to do so (no promise though).

File: resources/ui/outputhelpwin.ui [hunk 14]
```diff
@@ -34,7 +34,7 @@ p, li { white-space: pre-wrap; }
...
...ize:12pt;&quot;&gt;${desktopdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user desktop directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${downloaddir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user download directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${documentsdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user documents directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${musicdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user music directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${picturesdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user pictures directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${videosdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user video directory&lt;/span&gt;&lt;/p&gt;
```
Comment (SpotlightKid) [inline resources/ui/outputhelpwin.ui:37]: What exactly is the typo that was fixed here? It's very hard to see in this diff.

Comment (trebmuh) [reply resources/ui/outputhelpwin.ui:37]: _localized XDG user music directory_ -> _localized XDG user documents directory_

**Comment by SpotlightKid** [comment]
Unfortunately the branch of this PR is out-of-date again, because I merged #18 and maybe #17 was also merged after the last update of this branch.

I tried rebasing this branch on `master`, but I wasn't able to determine beyond doubt what the most current version of the changes should be. The diffs for these translation files can get very confusing.

Could you have another look, please?

Sorry about the poor coordination of the PRs.

**Comment by Houston4444** [comment]
No problem. that should be ok in #19 .

**Pull request closed without merging**